modules/otel-troubleshoot-metrics.adoc

```diff
@@ -19,26 +19,59 @@ The Operator creates a `<cr_name>-collector-monitoring` telemetry service that y
 
 .Procedure
 
-. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource:
+. Enable the telemetry service by adding the following lines in the `OpenTelemetryCollector` custom resource (CR):
 
 +
 [source,yaml]
 ----
+# ...
   config: |
     service:
       telemetry:
         metrics:
           address: ":8888" # <1>
+# ...
 ----
 <1> The address at which the internal collector metrics are exposed. Defaults to `:8888`.
 
-// TODO Operator 0.82.0 has spec.observability.metrics.enableMetrics config that creates ServiceMonitors for users
-
 . Retrieve the metrics by running the following command, which uses the port-forwarding Collector pod:
 +
 [source,terminal]
 ----
 $ oc port-forward <collector_pod>
 ----
 
-. Access the metrics endpoint at `+http://localhost:8888/metrics+`.
+. In the `OpenTelemetryCollector` CR, set the `enableMetrics` field to `true` to scrape internal metrics by using `PodMonitors` or `ServiceMonitors` depending on the deployment mode of the OpenTelemetry Collector:
+:
++
+[source,yaml]
+----
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+spec:
+# ...
+  mode: deployment
+  observability:
+    metrics:
+      enableMetrics: true # <1>
+# ...
+----
++
+[NOTE]
+====
+Alternatively, if you don't set the `enableMetrics` field to `true`, you can access the metrics endpoint at `+http://localhost:8888/metrics+`.
+====
+
+. In the *Observe* page in the web console, enable *User Workload Monitoring* to visualize the scraped metrics.
++
+[NOTE]
+====
+Not all processors expose the required metrics.
+====
+
+. In the *Observe* -> *Dashboards* page in the web console, select the *OpenTelemetry Collector* dashboard from the drop-down list to view it.
++
+[TIP]
+====
+You can filter the visualized data such as spans or metrics by the Collector instance, namespace, or OpenTelemetry components such as processors, receivers, or exporters.
```

Comment:
I think this sentence uses two sets of commas for what is technically two lists:

1. the Collector instance, namespace, or OpenTelemetry components
2. processors, receivers, or exporters (which are examples of the components in the first list)

In this case, I think it might be appropriate to use semicolons here as well to separate the different lists, as per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-semicolons#semicolons-between-items-in-a-series):
```suggestion
You can filter the visualized data such as spans or metrics by the Collector instance; namespace; or OpenTelemetry components such as processors, receivers, or exporters.
```

If I'm misinterpreting the grammar rule feel free to disregard this